src/main/java/org/altbeacon/beacon/service/ArmaRssiFilter.java

```diff
@@ -0,0 +1,52 @@
+package org.altbeacon.beacon.service;
+
+import org.altbeacon.beacon.logging.LogManager;
+
+/**
+ * This filter calculates its rssi on base of an auto regressive moving average (ARMA)
+ * It needs only the current value to do this; the general formula is  n(t) = n(t-1) - c * (n(t-1) - n(t))
+ * where c is a coefficient, that denotes the smoothness - the lower the value, the smoother the average
+ * Note: a smoother average needs longer to "settle down"
+ */
+public class ArmaRssiFilter implements RssiFilter {
+
+    private static int DEFAULT_ARMA_MEASUREMENT = -75;  //What is the typical RSSI that beacons are sending with?
+    private static double DEFAULT_ARMA_SPEED = 0.1;     //How likely is it that the RSSI value changes?
```

Comment:
This constant is probably a good value for applications where the mobile device is generally stationary relative to the beacon.  But in some applications, like indoor navigation, the Mobile device is typically moving at velocities of 1m/s.  For such applications, it is important to have the RSSI respond to updates more quickly, so a value of 1.0 may be more appropriate.  I don't have strong opinions about which should be the default, but the JavaDocs should probably explain this and suggest an alternate value for use cases where changes in RSSI need to come more quickly.

Another factor in how quickly the RSSI will change using this algorithm is the transmission frequency of the beacon.  Non-connectable beacons transmitting at 10Hz will allow RSSI changes 10x faster than ones transmitting at 1Hz.